e2e/pollers.go

```diff
@@ -27,14 +27,14 @@ const (
 	extractClusterParametersPollInterval    = 10 * time.Second
 	extractVMLogsPollInterval               = 10 * time.Second
 	getVMPrivateIPAddressPollInterval       = 5 * time.Second
-	waitUntilPodRunningPollInterval         = 5 * time.Second
-	waitUntilPodDeletedPollInterval         = 5 * time.Second
+	waitUntilPodRunningPollInterval         = 10 * time.Second
+	waitUntilPodDeletedPollInterval         = 10 * time.Second
 	waitUntilClusterNotCreatingPollInterval = 10 * time.Second
-	waitUntilNodeReadyPollingInterval       = 5 * time.Second
+	waitUntilNodeReadyPollingInterval       = 10 * time.Second
 
 	// Polling timeouts
 	vmssClientCreateVMSSPollingTimeout     = 10 * time.Minute
-	deleteVMSSPollingTimeout               = 5 * time.Minute
+	deleteVMSSPollingTimeout               = 10 * time.Minute
```

Comment:
im not sure increasing polling timeouts is what we wanna do here, ideally we'd poll for _less_ time so we can have a chance of retrying before the test times out

Reply:
probably fine for now since this is for deletion, though this entire "polling" framework/unit we've made here seriously needs to be revisited at some point

Reply:
Yeah I landed on 10, just because even with more retries it was just getting context exceeded.